src/assets/js/helpers.js

```diff
@@ -436,6 +436,7 @@ export default {
     let newVid = document.getElementById(partnerName + '-video') || document.createElement("video");
     newVid.id = `${partnerName}-video`;
     newVid.poster = "assets/images/poster.gif";
+    newVid.onplay = function(){this.poster = ""};
```

Comment:
This is an in between solution, we want to see the nice animation when we are waiting, but we don't need to see it when people use P.I.P. so once the video is playing we remove the poster to reduce painting calls.